scripts/ipfs-pi-stream/install

```diff
@@ -24,3 +24,14 @@ sudo systemctl enable process-stream
 
 # Add hourly job to clear out old data
 echo "41 * * * * $USER /usr/local/bin/ipfs repo gc" | sudo tee --append /etc/crontab
+
+# Install the ipfs video player
+mkdir "$BASE_DIR/tmp"
+git clone https://github.com/tomeshnet/ipfs-live-streaming.git "$BASE_DIR/tmp/live-streaming"
+cp -r "$BASE_DIR/tmp/live-streaming/terraform/shared/video-player" /var/www/html/video-player
```

Comment:
Do the `sed` in the `tmp` and copy at the end. This way we never get intermediate states in case something fails. If there is a file in `/var/www/html/video-player` we know the `sed` lines succeeded. This is also generally applicable.